client/ayon_maya/api/plugin.py

```diff
@@ -946,9 +946,11 @@ def update(self, container, context):
             cmds.sets(invalid, remove=node)
 
         # Update metadata
-        cmds.setAttr("{}.representation".format(node),
-                     repre_entity["id"],
-                     type="string")
+        for attr_name, value in {
+            "representation": repre_entity["_id"],
```

Comment:
```suggestion
            "representation": repre_entity["id"],
```

Reply:
Haha thanks - my IDE with copilot installed is starting to have a mind of its own sometimes.